src/mobile/src/components/Post/index.jsx

```diff
@@ -0,0 +1,122 @@
+import { StyleSheet, Text, View, useWindowDimensions } from 'react-native';
+import RenderHtml from 'react-native-render-html';
+import { Avatar } from 'react-native-elements';
+import * as WebBrowser from 'expo-web-browser';
+import useSWR from 'swr';
+import { MaterialIcons } from '@expo/vector-icons';
+
+import { tagsStyles, baseStyles } from './styles/post';
+import { formatPublishedDate } from '../../utils/formatPublishedDate';
+import { getNameInitial } from '../../utils/getNameInitial';
+import { createPostUrl } from '../../utils/createUrl';
+import Pagination from './Pagination';
+import Skeleton from '../Skeleton';
+
+const styles = StyleSheet.create({
+  postPage: {
+    marginVertical: 16,
+  },
+  title: {
+    fontSize: 24,
+    color: '#121D59',
+    fontWeight: 'bold',
+  },
+  postInfo: {
+    marginVertical: 12,
+    paddingVertical: 8,
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+  },
+  postErrorMessage: {
+    fontSize: 24,
+    color: '#EF4444',
+    fontWeight: 'bold',
+    textAlign: 'center',
+  },
+  authorInfo: {
+    color: '#9CA3AF',
+    marginLeft: 12,
+    flexShrink: 1,
+  },
+});
+
+const Post = ({ postURL, page, navigation }) => {
+  const { data: post, error: postError } = useSWR(postURL);
```

Comment:
Any idea why it makes so many request when I navigate to next/prev post? (~130 requests)
<img width="700" alt="image" src="https://user-images.githubusercontent.com/51073515/169610152-aeefaaf6-4549-4570-946f-c4a9aeea8d8f.png">

Reply:
Fixed adding by changing `SWRConfig` config